dist/angular-storage.js

```diff
@@ -94,7 +94,7 @@ angular.module('angular-storage.storage', [])
   }]);
 
 
-angular.module('angular-storage.store', ['angular-storage.internalStore'])
+angular.module('angular-storage.store', ['angular-storage.internalStore', 'ngCookies'])
```

Comment:
Don't add it here as a dependency. If you add it here as a dependency, it means that it MUST be present to start the app. Some people might not want the cookie fallback, and even if they don't, they have to add the `angular-cookies.js` if you add the module as a dependency. That's why, I'm using the `$injector` to get the dependency and not depending on `ngCookies`. Makes sense?